src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Event.java

```diff
@@ -23,18 +22,40 @@
     @Column(name = "EVENT_ID")
     private Long eventId;
 
+    @Column(name = "OFFENDER_ID")
+    private Long offenderId;
+
     @Column(name = "IN_BREACH")
     private Long inBreach;
 
     @Column(name = "NOTES")
     private String notes;
 
+    @Column(name = "EVENT_NUMBER")
+    private String eventNumber;
+
     @Column(name = "ACTIVE_FLAG")
     private Long activeFlag;
 
     @Column(name = "CONVICTION_DATE")
     private LocalDate convictionDate;
 
+    @Column(name = "REFERRAL_DATE")
+    private LocalDate referralDate;
+
     @Column(name = "SOFT_DELETED")
     private Long softDeleted;
+
+    @OneToOne(mappedBy = "event")
```

Comment:
All these back references explain why PR is much bigger than expected; much of the original work NR did for retrieving sentences, offences etc from court/institutional report can change from additional service calls to navigating the Delius model.

Reply:
ok